extension/source/background/QuillController.ts

```diff
@@ -53,9 +53,10 @@ export default class QuillController {
 
   constructor(
     public storage: CellCollection,
+    public encryptedStorage: CellCollection,
     public currencyConversionConfig: CurrencyConversionConfig,
   ) {
-    this.cells = QuillStorageCells(storage);
+    this.cells = QuillStorageCells(storage, encryptedStorage);
```

Comment:
considering these 2 are types of the same entity, we can group these and pass as an object instead of 2 different constructor arguments 

lets create a new type 
```ts
export type StorageConfig = {
  standardStorage: CellCollection;
  encryptedStorage: CellCollection;
};
```

and modify the constructor to accept a single storage object
```ts
public storage: StorageConfig,
```
and access standardStorage and encryptedStorage separately

```diff
- this.cells = QuillStorageCells(storage, encryptedStorage);
+ this.cells = QuillStorageCells(storage.standardStorage, storage.encryptedStorage);
```